src/BridgeMaker.js

```diff
@@ -7,7 +8,17 @@ const BridgeMaker = {
    * @param {function(): number} generateRandomNumber 무작위 값을 생성해주는 함수
    * @return {string[]} 입력받은 길이에 해당하는 다리 모양. 위 칸이면 U, 아래 칸이면 D로 표현해야 한다.
    */
-  makeBridge(size, generateRandomNumber) {},
+  makeBridge(size, generateRandomNumber) {
+    let solutionBridge = [];
+    for (let index = 0; index < size; index++) {
+      if (Number(generateRandomNumber())) {
+        solutionBridge.push(COMMAND.UP);
+        continue;
+      }
+      solutionBridge.push(COMMAND.DOWN);
+    }
```

Comment:
이 부분 저는 생성되는 랜덤 숫자(0 또는 1)에 따라서 true/false 값으로 판단해서 삼항연산자를 사용해봤는데
중복코드를 줄이는데에 도움이 되어서 남겨보고 갑니다 😊
```suggestion
    for (let index = 0; index < size; index++) {
      const number = generateRandomNumber();
      solutionBridge.push(number ? COMMAND.UP : COMMAND.DOWN);
    }
```

Reply:
자바스크립트 컨벤션에서 삼항연산자 쓰지 말라는 말을 봤던것?? 같았습니다!!!